android/app/src/main/java/com/reactnativeapp/MainActivity.java

```diff
@@ -43,6 +43,7 @@ private void setupEventSender() {
           new ArrayList<>(),
           BuildConfig.DEBUG,
           BuildConfig.DEBUG,
+          BuildConfig.DEBUG,
```

Comment:
I could not find info about this change in signature. Do you have any resource?

Reply:
Me neither, I had to search for it in mobile-android repo.